### DISCUSSION TITLE
Feature Suggestions

### DISCUSSION THREAD

**Discussion by WilliamGawlik** [WilliamGawlik/ac-F1-Regulations#46] [Ideas]
Use this discussion board to storm up new feature ideas.

**Comment by ovsyany** [comment]
Hello, William!
It would be nice in the future to be able to retire AI driver by button in the RARE Settings (permamently teleport into pitstop)
Easiest implementation is to give AI too much damage on one of his components.

I came up with this idea when I was looking at a damaged AI that was driving at half speed to the end of the course and started to interfere with the leaders. Another example is this AI pitstop-every-lap bug. If we could retire this cars, they wouldn't have to pitstop every lap and interfere with the race.

**Reply by WilliamGawlik** [reply]
Created the feature request https://github.com/ac-sim-improvement-mods/ac-rules-and-race-enhancements/issues/55

**Comment by avjterava** [comment]
Dear developer!
Would be nice if in the future we could pre-determine what tyre compound AI will choose in the pits. That would be very convenient when using Pure Dynamic Weather feature. Lets say, there will be rain in the middle part of race and we definitely want AI to choose wet compound when going to pits. Thank You for your nice app! This changed everything!
Veka Terävä
